hack/flags.sh

```diff
@@ -81,6 +81,11 @@ parse_flags() {
   fi
   export ROOT_WORKSPACE=${ROOT_WORKSPACE:-"root"}
 
+  # Convert home ROOT_WORKSPACE to full path
+  if [ "$ROOT_WORKSPACE" == "~" ]; then
+    ROOT_WORKSPACE=$(KUBECONFIG=${KCP_KUBECONFIG} kubectl ws '~' --short)
```

Comment:
we need to export it, right?
```suggestion
    export ROOT_WORKSPACE=$(KUBECONFIG=${KCP_KUBECONFIG} kubectl ws '~' --short)
```

Reply:
not needed, it's already exported on line 82